roles/splunk_common/tasks/enable_forwarding.yml

```diff
@@ -51,22 +51,19 @@
   when:
     - splunk_indexer_cluster | bool
     - item.value | length > 0
-  no_log: "{{ hide_password }}"
   register: default_tcpout_group
 
 # NOTE: If this task is called or used, it will disable all local indexing!
 - name: Disable indexing on the current node
   ini_file:
     path: "{{ splunk.home }}/etc/system/local/outputs.conf"
     section: "indexAndForward"
-    option: "{{ item.key }}"
-    value: "{{ item.value }}"
+    option: "index"
```

Comment:
Why did you change this from the withitems loop?  Because these options will never expand or change?

Reply:
Yeah, I figured we weren't really using the loop very much with just 1 item. The `indexAndForward` section only exposes 2 possible keys anyways: https://docs.splunk.com/Documentation/Splunk/8.0.6/Admin/Outputsconf#IndexAndForward_Processor-----